ARC specification.md

```diff
@@ -232,6 +233,37 @@ The file `arc.cwl` MUST exist at the root directory of each ARC. It describes wh
 
 `arc.cwl` MUST be a CWL v1.2 workflow description and adhere to the same requirements as [run descriptions](#run-description). In particular, references to study or assay data files, nested workflows MUST use relative paths. An optional file `arc.yml` MAY be provided to specify input parameters.
 
+### Data Path Annotation
+
+All metadata references to files or directories located inside the ARC MUST follow the following patterns:
+
+- The `general pattern`, which is always applicable and SHOULD always be used is to specify the path relative to the ARC root
```

Comment:
Maybe rephrase the "which is always applicable". 
Even though the SHOULD is clearly not a MUST, to me the "always" sounds like this must always be done, which would contradict below folder-specific pattern.

Reply:
Thanks for you input! 
IMO the `SHOULD` is really a SHOULD as the `general pattern` is way more verbose so it probably came off exactly like planned. But I still softened it up a bit. Does this match more what you would expect now?

PS: Ho Ho Ho! 🎅